gluoncv/model_zoo/siamrpn/siam_rpn.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from mxnet.gluon.block import HybridBlock
+from mxnet.gluon import nn
+import mxnet as mx
+
+class RPN(nn.HybridBlock):
+    def __init__(self):
+        super(RPN, self).__init__()
+
+    def forward(self, z_f, x_f):
+        raise NotImplementedError
+
+class DepthwiseXCorr(HybridBlock):
+    def __init__(self,in_channels,hidden,out_channels, kernel_size=3):
+        super(DepthwiseXCorr, self).__init__()
+        self.conv_kernel = nn.HybridSequential(prefix='')
+        self.conv_search = nn.HybridSequential(prefix='')
+        self.head = nn.HybridSequential(prefix='')
+
+        self.conv_kernel.add(
+                nn.Conv2D(hidden, kernel_size=kernel_size,use_bias=False),
+                nn.BatchNorm(),
+                nn.Activation('relu')
+                )
+        self.conv_search.add(
+                nn.Conv2D(hidden, kernel_size=kernel_size,use_bias=False),
+                nn.BatchNorm(),
+                nn.Activation('relu')
+                )
+        self.head.add(
+                nn.Conv2D(hidden, kernel_size=1,use_bias=False),
+                nn.BatchNorm(),
+                nn.Activation('relu'),
+                nn.Conv2D(out_channels, kernel_size=1)
+                )
+        
+
+    def hybrid_forward(self,F, kernel, search):
+        kernel = self.conv_kernel(kernel)#[1, 256, 24, 24]
+        search = self.conv_search(search)#[1, 256, 4, 4]
+        batch = kernel.shape[0]
```

Comment:
In hybrid mode, we can't use `.shape` get shape anymore. We can discuss offline how to bypass this issue.